gazebo_ros/test/test_plugins.cpp

```diff
@@ -80,7 +80,7 @@ INSTANTIATE_TEST_CASE_P(
         "worlds/ros_world_plugin.world"}, {"test"}}),
     TestParams({{"-s", "libgazebo_ros_init.so", "worlds/sdf_node_plugin.world"}, {"/foo/my_topic"}})
     // cppcheck-suppress syntaxError
-  ), );
+  ));
```

Comment:
I'm just guessing that this change is what is responsible or the [uncrustify errors](http://build.ros2.org/job/Rpr__gazebo_ros_pkgs__ubuntu_focal_amd64/23/)